app/modules/enduser_setup.c

```diff
@@ -105,12 +105,26 @@ static const char http_html_gzip_contentencoding[] = "Content-Encoding: gzip\r\n
 /* Externally defined: static const char enduser_setup_html_default[] = ... */
 #include "enduser_setup/enduser_setup.html.gz.def.h"
 
+#define SCAN_LISTENER_MAGIC  1
+#define HTTP_REQUEST_BUFFER_MAGIC  2
```

Comment:
How about adding a comment to these two to describe their purpose - or giving them an expressive name?  "magic" could mean anything. Same goes for the `magic` variables in the structs below.

Reply:
Done